README.md

```diff
@@ -98,11 +101,6 @@ Keep in mind that companies may have job postings on their own site that are not
 
 #### [Flytographer](http://www.flytographer.com/) (👩‍💻 Co-Op / Intern Friendly)
 
-#### [Freshworks Studio](https://freshworks.io/)
```

Comment:
We need to leave the header section for each business - unless they've stopped operating their website/business entirely. Please only remove broken job links, thanks!